frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -429,7 +429,7 @@ const MessageInput = (props: Props) => {
                 ref={textAreaRef}
                 rows={1}
                 name="inputBar"
-                placeholder={channelConnected && 'Enter a message...'}
+                placeholder={channelConnected ? 'Enter a message...' : ''}
                 autoFocus={channelConnected}
```

Comment:
this was creating an error: when the channel was disconnected, the line was evaluated as false